### ISSUE TITLE
Catalog Prototype - bad url for VoiceOver Take 5

### ISSUE THREAD

**Issue by andrewpmiller** [gymnasium/tracker#198]
https://thegymnasium.com/dev#accessibility

The "Watch Now" link for **Introduction to Screen Readers Using VoiceOver** is not correct — it's missing the `/courses/take5/` part.
